std/json.d

```diff
@@ -456,15 +490,18 @@ struct JSONValue
     ///
     /// Throws: $(D JSONException) if $(D type) is not $(D JSON_TYPE.OBJECT)
     /// or $(D JSON_TYPE.NULL).
-    void opIndexAssign(T)(auto ref T value, string key) pure
+    void opIndexAssign(T)(auto ref T value, string key) pure @safe
```

Comment:
As above, and also to advertise to people that they can in fact use opIndexAssign from @safe code.